### PR TITLE
Fixed Minor Errors in header buttons and on chevron icon

### PR DESCRIPTION
**Pull request by bigbluebroccoli**
# Overview

Fixed errors with header buttons not seeming interactive and fixed issue with avatar.tsx page

# Tickets

- [Fix p is a descendant of div in Avatar.tsx](https://app.asana.com/0/1205185701134472/1205185248260816/f)
- [My Profile and AstroNinjas buttons cursor effect on hover](https://app.asana.com/0/1205185701134472/1205185389993812/f)
- [Fix linting issues + other minor errors.](https://app.asana.com/0/1205185701134472/1205201795207398/f)

# Screenshots

**Before**
N/A

**After**
N/A

# Testing

Tested on all screen sizes using Chrome Developer tools and no visual errors all worked as expected.

### CODE REVIEW

**Review by nurs-jubatyrov — COMMENTED** [review]
Is this PR still draft or ready for review? 

If it's ready, can you make sure to attach screenshot/video to show your fixes and also update PR description with updates you've made. 

Also, why package-lock.json has changed?

**Comment by bigbluebroccoli** [comment]
> Is this PR still draft or ready for review?
> 
> If it's ready, can you make sure to attach screenshot/video to show your fixes and also update PR description with updates you've made.
> 
> Also, why package-lock.json has changed?

Unsure why package-lock.json has changed as I didnt make any edits to it, I will have a look now and change it back to its original state.

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "@tzkt/sdk-api": "^2.2.0",
     "eslint-config-next": "^12.0.4",
     "framer-motion": "^10.12.16",
+    "lint": "^0.8.19",
```
Comment (nurs-jubatyrov) [inline package.json]: What's the purpose of lint here? If it's used for linting then it should be under "devDependency".

File: pages/purchase.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ export default function Home() {
         const data = await response.json();
         setBalance(data / 1000000);
 
-        if (data < 1) {
+        if (data / 1000000 < 1) {
```
Comment (nurs-jubatyrov) [inline pages/purchase.tsx]:
```
const MUTEZ = 1000000;
const balanceInMutez = await response.json();
const balance = balanceInMutez / Mutez
setBalance(balance) 

if (balance < 1) 
```

File: pages/purchase.tsx [hunk 3]
```diff
@@ -40,10 +40,12 @@ export default function Home() {
         const response = await fetch(
           `https://api.tzkt.io/v1/accounts/${address}/balance`
         );
-        const data = await response.json();
-        setBalance(data / 1000000);
+        const MUTEZ = 1000000;
+        const MUTEZBalance = await response.json();
+        const FinalBalance = MUTEZBalance / MUTEZ;
+        setBalance(FinalBalance);
```
Comment (nurs-jubatyrov) [inline pages/purchase.tsx]: Thanks Daniel for update, there are some further updates I'd like you to address. 

I want you to go https://www.syncfusion.com/blogs/post/10-javascript-naming-conventions-every-developer-should-know.aspx and verify whether your variable names satisfy the conventions.

**Review by nurs-jubatyrov — APPROVED** [review]

**Pull request merged**